Cargo.toml

```diff
@@ -10,6 +10,8 @@ serde = "1"
 indexmap = { version = "2", optional = true }
 hashbrown = { version = "0.14", optional = true }
 unic-segment = {version = "0.9", optional = true}
+itoa = "1.0.9"
+ryu = "1.0.15"
```

Comment:
Let's avoid adding dependencies. Does it make a big difference?

Reply:
I'll check, but most likely yes. If we don't add these we fall back to `std::fmt`

Reply:
Yup, `std::fmt` is (3%, 5%, and 21%) slower (on some tests and no difference in the rest):

```log
     Running benches/iai-templates.rs (target/release/deps/iai_templates-17599881302f6267)
iai_templates::templates::render teams:teams_setup()
  Instructions:               19934|21063           (-5.36011%) [-1.05664x]
  L1 Hits:                    28499|30201           (-5.63557%) [-1.05972x]
  L2 Hits:                       18|28              (-35.7143%) [-1.55556x]
  RAM Hits:                     320|345             (-7.24638%) [-1.07812x]
  Total read+write:           28837|30574           (-5.68130%) [-1.06024x]
  Estimated Cycles:           39789|42416           (-6.19342%) [-1.06602x]
iai_templates::templates::render big_table:big_table_setup()
  Instructions:             6844032|8726031         (-21.5676%) [-1.27498x]
  L1 Hits:                 10283883|13175948        (-21.9496%) [-1.28122x]
  L2 Hits:                    12264|12275           (-0.08961%) [-1.00090x]
  RAM Hits:                    1984|2005            (-1.04738%) [-1.01058x]
  Total read+write:        10298131|13190228        (-21.9261%) [-1.28084x]
  Estimated Cycles:        10414643|13307498        (-21.7385%) [-1.27777x]
iai_templates::templates::render realistic:realistic_setup()
  Instructions:              118572|122377          (-3.10924%) [-1.03209x]
  L1 Hits:                   164464|170207          (-3.37413%) [-1.03492x]
  L2 Hits:                      393|435             (-9.65517%) [-1.10687x]
  RAM Hits:                     625|651             (-3.99386%) [-1.04160x]
  Total read+write:          165482|171293          (-3.39243%) [-1.03512x]
  Estimated Cycles:          188304|195167          (-3.51648%) [-1.03645x]
iai_templates::templates::context big_context:big_context_setup()
  Instructions:              417874|418633          (-0.18130%) [-1.00182x]
  L1 Hits:                   581471|582630          (-0.19893%) [-1.00199x]
  L2 Hits:                     4583|4603            (-0.43450%) [-1.00436x]
  RAM Hits:                     189|179             (+5.58659%) [+1.05587x]
  Total read+write:          586243|587412          (-0.19901%) [-1.00199x]
  Estimated Cycles:          611001|611910          (-0.14855%) [-1.00149x]
     Running benches/iai-value.rs (target/release/deps/iai_value-dabe104d28c4eb05)
iai_value::serialize::serialize_value page:& Page :: default()
  Instructions:                6741|6766            (-0.36949%) [-1.00371x]
  L1 Hits:                     9236|9265            (-0.31301%) [-1.00314x]
  L2 Hits:                        8|7               (+14.2857%) [+1.14286x]
  RAM Hits:                     136|140             (-2.85714%) [-1.02941x]
  Total read+write:            9380|9412            (-0.33999%) [-1.00341x]
  Estimated Cycles:           14036|14200           (-1.15493%) [-1.01168x]
```

Reply:
To test this I made the deps optional and added a `no-fmt` feature. I could push that as a compromise.

Reply:
I'm thinking of having a `speedup` feature or something rather than `n` features to turn on for things that are purely internal and that won't really matter to the end user. I'll run both on my machine as well to see before merging. I'm still puzzled why i don't get any perf diff with the hashbrown feature 🤔